functions/src/models/metrics.ts

```diff
@@ -69,3 +71,19 @@ export const addFeedback = async (feedback: Feedback) => {
       createdAt: Timestamp.now(),
     });
 };
+
+export const getFeedbackByExercise = async (
+  exerciseId: string,
+  mode?: SessionMode,
+) => {
+  let query = firestore()
+    .collection(FEEDBACK_COLLECTION)
+    .where('exerciseId', '==', exerciseId);
+
+  if (mode) {
+    query = query.where('mode', '==', mode);
+  }
```

Comment:
Do we have an index for this combo?

Reply:
Good reminder! haven't added to the indexes yet! will do it now 👍